src/contracts/pods/EigenPod.sol

```diff
@@ -454,24 +508,25 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         // Withdrawal credential proofs should only be processed for "INACTIVE" validators
         require(
             validatorInfo.status == VALIDATOR_STATUS.INACTIVE,
-            "EigenPod.verifyCorrectWithdrawalCredentials: Validator must be inactive to prove withdrawal credentials"
+            "EigenPod._verifyWithdrawalCredentials: validator must be inactive to prove withdrawal credentials"
         );
 
-        // Ensure the `validatorFields` we're proving have the correct withdrawal credentials
+        // Validator should not already be in the process of exiting
+        require(
+            validatorFields.getExitEpoch() == FAR_FUTURE_EPOCH,
+            "EigenPod._verifyWithdrawalCredentials: validator must not be exiting"
+        );
+
+        // Ensure the validator's withdrawal credentials are pointed at this pod
         require(
             validatorFields.getWithdrawalCredentials() == bytes32(_podWithdrawalCredentials()),
```

Comment:
oh, that's a good idea! yeah, we could do that. will add it to a list of features to discuss in the PR comment above.

Reply:
yep, i'm leaning towards keeping this as-is (check for 0x01 creds), because i think we're likely to do a small patch before Pectra anyway to prep for that upgrade.

we can add this at that time.